sqlalchemy_ibmi/base.py

```diff
@@ -400,10 +404,21 @@ def visit_match_op_binary(self, binary, operator, **kw):
             self.process(binary.left),
             self.process(binary.right))
 
+    # This is to allow legacy LIMIT replacement.
+    #LIMIT is acceptable since IBM i 7.1 TR11,IBM i 7.2 TR3 
     def limit_clause(self, select, **kwargs):
+        custom_legacy_limit_option = self.dialect.use_legacy_limit
+        if custom_legacy_limit_option is not None:
+            custom_legacy_limit_option = custom_legacy_limit_option.lower() == 'true'
```

Comment:
TBH, IBM i 7.1 TR11 and 7.2 TR3 are old enough I don't think we need an option here. We can make that the minimum supported IBM i versions.

Reply:
I personally did not need it. Figured it might be needed to accept this PR. I can remove it.